frontend/src/App.tsx

```diff
@@ -26,16 +41,18 @@ export function App() {
       <CustomProvider locale={frFR}>
         <Provider store={homeStore}>
           <PersistGate loading={undefined} persistor={persistor}>
+            <RouterProvider fallbackElement={<p>Chargement...</p>} router={router} />
+            {/* 
+            
+            2ème possibilité ici : 
             <Router>
-              <Switch>
-                <Route exact path="/side_window">
-                  <SideWindow />
-                </Route>
-                <Route path="/">
-                  <HomePage />
-                </Route>
-              </Switch>
-            </Router>
+              <Routes>
```

Comment:
Je préfère ce pattern qui correspond à ce qui est fait in fine quand on a des routes nestées il me semble.